views/layout.jade

```diff
@@ -20,6 +20,7 @@ html
     script(src='https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js')
     script(src='/javascripts/bootstrap.js')
     script(src='/javascripts/nfl-data.js')
+    script NFLData.USE_CACHED_PLAYER_DATA = #{USE_CACHED_PLAYER_DATA || false};
```

Comment:
Not sure if this is the best way to set the flag in client JS from an env variable. Let me know if you want me to change it.